opennms-webapp/src/main/webapp/hardware/list.jsp

```diff
@@ -43,9 +43,67 @@
                 org.springframework.web.context.WebApplicationContext,
                 org.springframework.web.context.support.WebApplicationContextUtils"
 %>
+<%@ page import="org.opennms.netmgt.model.OnmsMetaData" %>
+<%@ page import="org.opennms.netmgt.model.snmpmetadata.SnmpMetadataEntry" %>
+<%@ page import="org.opennms.netmgt.model.snmpmetadata.SnmpMetadataObject" %>
+<%@ page import="org.opennms.netmgt.model.snmpmetadata.SnmpMetadataTable" %>
+<%@ page import="org.opennms.netmgt.model.snmpmetadata.SnmpMetadataBase" %>
+<%@ page import="org.opennms.netmgt.model.snmpmetadata.SnmpMetadataValue" %>
 <%@taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
 <%!
+    public void printSnmpMetadata(final JspWriter out, final SnmpMetadataBase object) throws Exception {
```

Comment:
Split to 4 methods with concrete type for param `object`

Reply:
done.